app/controllers/nsm/claims_controller.rb

```diff
@@ -43,6 +44,12 @@ def destroy
       redirect_to draft_nsm_applications_path, flash: { success: t('.deleted') }
     end
 
+    def clone
+      clone_application.save
+
+      redirect_to draft_nsm_applications_path, flash: { success: t('.cloned') }
```

Comment:
If you have hundreds of drafts it might be a kindness to identify which is the new clone - perhaps taking us to the start page for that claim? Or putting its LAA reference in the flash?

Reply:
I was leaning towards the former at first but then that felt like guessing what you wanted to do. I guess most of the time you'll want to clone then submit, so going to the start page probably makes the most sense.